Models/Queue.js

```diff
@@ -217,15 +218,15 @@ export class Queue {
    * worker function that has concurrency X > 1, then X related (jobs with same name)
    * jobs will be returned.
    *
-   * If queue is running with a lifespan, only jobs with timeouts at least 500ms < than REMAINING lifespan
-   * AND a set timeout (ie timeout > 0) will be returned. See Queue.start() for more info.
-   *
    * @param queueLifespanRemaining {number} - The remaining lifespan of the current queue process (defaults to indefinite).
    * @return {promise} - Promise resolves to an array of job(s) to be processed next by the queue.
    */
   async getConcurrentJobs(queueLifespanRemaining = 0) {
```

Comment:
this function has grown quite a bit. Consider breaking it up into multiple functions that break up the different logistical steps